cmd/experimental/mirror/posix/main.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2025 The Tessera authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// mirror/posix is a command-line tool for mirroring a tlog-tiles compliant log
+// into a POSIX filesystem.
+package main
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"net/url"
+	"os"
+	"path/filepath"
+
+	"github.com/transparency-dev/trillian-tessera/client"
+	mirror "github.com/transparency-dev/trillian-tessera/cmd/experimental/mirror/internal"
+	"k8s.io/klog/v2"
+)
+
+var (
+	storageDir = flag.String("storage_dir", "", "Root directory to store log data.")
+	sourceURL  = flag.String("source_url", "", "Base URL for the source log.")
+	numWorkers = flag.Uint("num_workers", 30, "Number of migration worker goroutines.")
+)
+
+func main() {
+	klog.InitFlags(nil)
+	flag.Parse()
+	ctx := context.Background()
+
+	srcURL, err := url.Parse(*sourceURL)
+	if err != nil {
+		klog.Exitf("Invalid --source_url %q: %v", *sourceURL, err)
+	}
+	src, err := client.NewHTTPFetcher(srcURL, nil)
```

Comment:
Does this imply the `mirror/posix` command doesn't support mirroring another posix log? (although it's trivial to copy and paste the directory)